docs/guides/developer/docs/api/security-api.md

```diff
@@ -30,21 +30,43 @@ Even though this greatly increases security in sense that signed URLs can only b
 
 Returns a signed URL that can be played back for the indicated period of time, while access is optionally restricted to the specified IP address.
 
-Form Parameters            |Type            | Description
-:--------------------------|:---------------|:----------------------------
-`url`                      | String         | The linke to encode. **required**
-`valid-until`              | date           | Until when is the signed url valid
-`valid-source`             | ip             | The IP address from which the url can be accessed
-
+Form Parameters |Required |Type                                  | Description
+:---------------|:--------|:-------------------------------------|:----------------------------
+`url`           |yes      | [`string`](types.md#basic)           | The URL to be singned
+`valid-until`   |no       | [`datetime`](types.md#date-and-time) | Until when is the signed url valid
```

Comment:
Until when is the signed url valid would read better as When the signed url expires